features/src/main/resources/schema/features/openapi.yaml

```diff
@@ -592,6 +592,8 @@ components:
           type: string
           description: a description of the features in the collection
           example: An address.
+        summaries:
```

Comment:
these files download from ogcapi website and should not change its definition, please rollback changes to this file